metricbeat/module/mongodb/metrics/_meta/fields.yml

```diff
@@ -0,0 +1,415 @@
+- name: metrics
+  type: group
+  description: >
+    Statistics that reflect the current use and state of a running `mongod` instance
+    for more information, take a look at https://docs.mongodb.com/manual/reference/command/serverStatus/#serverstatus.metrics
+  fields:
```

Comment:
Could you add descriptions to fields too?

Reply:
Done